model/clusters_mgmt/v1/machine_pool_type.model

```diff
@@ -16,18 +16,23 @@ limitations under the License.
 
 // Representation of a machine pool in a cluster.
 class MachinePool {
-	// ID used to identify the cluster that this machinepool is attached to.
-	link Cluster Cluster
+  // ID used to identify the cluster that this machinepool is attached to.
+  link Cluster Cluster
```

Comment:
There seems to be some misalignment in the formatting of indentation.
In the other file, there's 4 spaces leading to comment, and a tab for the fields.
Here' it's 2 spaces for both comment and field.

Reply:
Indeed. Not sure why this happened.
How should it be?

Reply:
Fixed. RePTAL